<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionCalcDialog</class>
 <widget class="QDialog" name="OptionCalcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Option Pricing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Set as default</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pricingPushButton">
     <property name="text">
      <string>Pricing</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="classCodeLabel">
          <property name="text">
           <string>Class Code</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="classCodeComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="optionTypeLabel">
          <property name="text">
           <string>Option Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="optionTypeComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="volatilityLabel">
          <property name="text">
           <string>Volatility</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="volatilityLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="yieldRateLabel">
          <property name="text">
           <string>Yield Rate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="yieldRateLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="freeRateLabel">
          <property name="text">
           <string>Free Rate</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="freeRateLineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="strikePriceLabel">
          <property name="text">
           <string>Strike Price</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="strikePriceLineEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="spotPriceLabel">
          <property name="text">
           <string>Spot Price</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="spotPriceLineEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="maturityLabel">
          <property name="text">
           <string>Maturity</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QDateEdit" name="maturityDateEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="priceLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="priceTextLabel">
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="getDefaultPushButton">
     <property name="text">
      <string>Get default values</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
